### PR TITLE
Deployment improvements

### PR DESCRIPTION
**Pull request by iloveitaly**
useful when deploying to dokku

### CODE REVIEW

File: config/unicorn.rb [hunk 1]
```diff
@@ -1,13 +1,16 @@
 worker_processes ENV.fetch('WORKER_PROCESSES', 5).to_i
 timeout 180
 
+if File.writable?("/var/log/unicorn.stderr.log")
+  stderr_path "/var/log/unicorn.stderr.log"
+  stdout_path "/var/log/unicorn.stdout.log"
+end
```
Comment (phstc) [inline config/unicorn.rb]: does that work with Heroku?

File: config/unicorn.rb [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+
 preload_app true
 
 GC.respond_to?(:copy_on_write_friendly=) and  GC.copy_on_write_friendly = true
 
 before_fork do |server, worker|
-  defined?(ActiveRecord::Base) and ActiveRecord::Base.connection.disconnect!
```
Comment (phstc) [inline config/unicorn.rb]: why no `disconnect!` as suggested here: https://devcenter.heroku.com/articles/rails-unicorn#config ?

Comment (iloveitaly) [inline config/unicorn.rb]: I don't believe ActiveRecord is used in this endpoint, so we don't need this line.

File: config/unicorn.rb (hunk 1 above)
Comment (iloveitaly) [inline config/unicorn.rb]: I believe so: I remember testing this idiom a while back and it worked on Heroku. I'll double check and get back to you.

File: config/unicorn.rb (hunk 2 above)
Comment (phstc) [inline config/unicorn.rb]: :scream: you are more than right! good point hehe

File: config/unicorn.rb (hunk 1 above)
Comment (phstc) [inline config/unicorn.rb]: Running on a Heroku app `heroku run bundle exec rails c`:

```ruby
irb(main):001:0> File.writable?("/var/log/unicorn.stderr.log")
=> false
```

Comment (iloveitaly) [inline config/unicorn.rb]: Perfect: then this should work just as designed. Heroku will output to stdout but when hosting dokku you can output to a file. This is useful for piping that file to PaperTrail.

**Pull request merged**